package.json

```diff
@@ -25,11 +25,10 @@
     "lib": "lib"
   },
   "dependencies": {
-    "appium-base-driver": "^2.0.1",
+    "babel-runtime": "=5.8.24",
```

Comment:
Can you put this in alphabetical order? If it is not it will just be done next time someone uses an automatic npm command, and will add an erroneous entry to the diff there.